angular-ngrx-scss/src/app/home/user-gists/user-gists.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+  <h3>Gists</h3>
+  <ng-container *ngIf="userGists$ | async as gists">
+    <ng-container *ngIf="gists.length > 0; else noGists">
+      <div class="list-container">
+        <ng-container *ngFor="let gist of gists">
+          <a class="link" [href]="gist.url" target="_blank">{{gist.fileName}}</a>
+        </ng-container>
+      </div>
+    </ng-container>
+  </ng-container>
+</div>
+
+<ng-template #noGists>
```

Comment:
I'm.....not sure we actually need this check? I think if there's no gists we can just leave the section blank. But we'll want to check one of the completed showcases (Angular/Apollo or Next/React-Query) from someone's account that doesn't have any gists. What do those showcases do in that case? 

- [ ] Please verify this on a completed showcase with no user gists, see if they show a message or just leave the aside blank